editor/utils/test/terms.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Internal dependencies
+ */
+import { buildTermsTree } from '../terms';
```

Comment:
Do we really need the test twice?

Reply:
In case it diverges, it might make sense.